pages/admin/request/create-renewal.tsx

```diff
@@ -31,8 +31,12 @@ import {
   GetApplicantRenewalResponse,
 } from '@tools/pages/admin/requests/types';
 import { useRouter } from 'next/router';
+import BackToSearch from '@components/admin/requests/modals/BackToSearchModal';
+import { ApplicantData } from '@tools/pages/admin/permit-holders/types';
+import SelectedPermitHolderCard from '@components/admin/requests/SelectedPermitHolderCard';
 
 export default function CreateRenewal() {
+  const [onRequestPage, setOnRequestPage] = useState<boolean>(false);
```

Comment:
This variable name isn't clear what it does. Try to make it more clear as to its functionality, even if it means the variable name is longer. If that isn't possible, add a comment

Reply:
Also, it would be better to represent the current step using an enum with 2 values. It is easier to read/understand (since steps are numeric and sequential) than a boolean which can only support 2 steps